crates/rolldown_common/src/module_loader/task_result.rs

```diff
@@ -12,4 +14,5 @@ pub struct NormalModuleTaskResult {
   pub warnings: Vec<BuildDiagnostic>,
   pub module: Module,
   pub ecma_related: Option<(EcmaAst, SymbolRefDbForModule)>,
+  pub dynamic_import_rec_exports_usage: FxHashMap<ImportRecordIdx, DynamicImportExportsUsage>,
```

Comment:
This is temp data structure, it will be normalized to `FxHashmap<ModuleId, DynamicUsage>` after fetching all modules, I don't see any usage related to specific modules, so I don't add this field to `ecma_view`. Also only those modules are used as a dynamic entry will have a none empty field, for most of the modules this field is just a waste of memory.

Reply:
> so I don't add this field to ecma_view.

`ecma_related` is not `ecma_view`.

> for most of the modules this field is just a waste of memory.

I'm thinking the same with you. Most of the module tasks don't need to create `dynamic_import_rec_exports_usage: FxhashMap` .

Reply:
> Maybe put this in `ecma_related` fields?

looks good, will refactor it in next pr.